Project.toml

```
@@ -42,6 +43,7 @@ Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 
 [compat]
+Adapt = "3.7"
```

Why this version?

Would this work?
```suggestion
Adapt = "3, 4"
```

---

This won't do anything yet because RRTMGP still restricts us

---

Yeah my intention was to pin it to a known working version, here, `3.7` (given the issues we seemed to be having elsewhere with CUDA / Adapt upgrades)